docs/grafana-ui.rst

```diff
@@ -0,0 +1,37 @@
+Grafana-UI
+==========
+
+
+.. todo::
+
+  The following test has been realized on vagrant only - One should list
+  all requirements for reaching the UI using a operationnal cluster.
+
+  On your host, you will need to install kubectl.
+
+
+On your host, connect to the vagran VM
```

Comment:
missing `t` at `Vagrant` (also capitalize the first `v`).

Reply:
correction done.